backend/coreapp/serializers.py

```diff
@@ -63,6 +63,30 @@ class LibrarySerializer(serializers.Serializer[Library]):
     version = serializers.CharField()
 
 
+class PresetNameSerializer(serializers.ModelSerializer[Preset]):
```

Comment:
for the sake of consistency, I think it'd be good to have both of these be X PresetSerializer. this one could be TinyPresetSerializer or NamePresetSerializer or something like that